primefaces/src/main/java/org/primefaces/component/commandbutton/CommandButtonRenderer.java

```diff
@@ -174,7 +174,7 @@ protected void encodeScript(FacesContext context, CommandButton button) throws I
         WidgetBuilder wb = getWidgetBuilder(context);
         wb.init("CommandButton", button)
             .attr("disableOnAjax", button.isDisableOnAjax(), true)
-            .attr("disabled", button.isDisabled(), false);
+            .attr("disabledAttr", button.isDisabled(), false);
```

Comment:
`this.cfg.disabled` was already used in `SplitButton` and `MenuButton`, so I opted to rename it on each component for consistency.